libs/wingsdk/src/target-tf-aws/bucket.ts

```diff
@@ -181,6 +181,13 @@ export function createEncryptedBucket(
   });
 
   if (isPublic) {
+    new S3BucketPublicAccessBlock(scope, "PublicAccessBlock", {
```

Comment:
Why do we need a resource named S3BucketPublicAccessBlock when the bucket is supposed to be public? Maybe we can add a comment to explain why its needed in the case of `cloud.Website`

Reply:
When we create a bucket it's always blocked,  to give access we need to remove this blocked then create the access policy

If we not open the bucket the policy it's not created at all and break the deploy because it's not permitted give access to a blocked bucket

Reply:
Oh I see - sounds like it might be related to this https://aws.amazon.com/about-aws/whats-new/2022/12/amazon-s3-automatically-enable-block-public-access-disable-access-control-lists-buckets-april-2023/